### DISCUSSION TITLE
MIB2 Harman conversion from VW/Skoda to Audi

### DISCUSSION THREAD

**Discussion by em1ter** [Mr-MIBonk/M.I.B._More-Incredible-Bash#33] [Q&A]
Hy everyone,

First of all, thank you all for your efforts on creating this magic tool! You are doing a great job!
I've been messing a little with MIB2 STD but recently have changed my car to Audi with failing MHIG and planning to install MHI2 there, so will definitely have the opportunity to check this tool in action.
But due to some specific issues I'm not able to get a direct replacement of my 8V0 unit to 8V1 MIB2 unit that fits my car directly. So my only option is to convert VW/Skoda/Seat 3G0/3V0/5F0 units to 8V1 Audi A3. That's why I need your help answering few of my questions:
1) Is it possible at all? I've read about some conversions from Skoda to VW, from VW to Seat etc, but haven't seen any descriptions on how it's done. Just a brief mention about train should be modified in EEPROM.
2) Some advises on how to do that. I do not require a full step by step manual, but at least some words on what should be done first and maybe sharing your experience with some examples of conversion between Skoda/VW/Seat.
3) My additional concern is about the controls of the unit. I assume that converted unit will support my Audi stock controls (hardware switches and rotary puck), but what about non converted unit? I mean, Skoda/VW/Seat use touch screen to control the unit, but will Audi switches and puck control Skoda/VW/Seat unit in plug and play manner without any patching? Asking because I'm quite sure that there are actions that have to be performed in the unit using its own control organs like starting the update and if Audi controls fail to run through Skoda/VW unit - I will stuck on that point. Have someone experience running VW/Skoda units with Audi screen/controls?

Appreciate your answers in advance!

**Comment by harman-f** [comment]
8V1 is Audi A3.
I'm not 100% familiar with Audi hardware, but if I remember correctly A3 has a few more connectors on the back of the unit than most other MHI2 units on other brands.
![grafik](https://user-images.githubusercontent.com/98130152/188020986-87b610ef-06de-44aa-8819-173e93457f1a.png)
This makes this unit extra expensive ;)

4M1 (Audi Q7) should work, as they also have the full set of connectors
Same goes for 8S0 (Audi TT)

Otherwise, conversion between brands is no problem.

**Reply by em1ter** [reply]
harman-f you are right.
8V1 Audi A3 MIB2 units have these 2 extra LVDS connectors on its back. As far as I have researched the cream one is used for Audi TopView system (or TV tuner, according to some other sources) and the light green one is used to communicate with Virtual cockpit (some Audi models have additional LVDS beside MOST and regular CAN). 8S turns out to be pretty unique not having a main display unit but using it's cockpit for these purposes.
All Audi models in general use these specific MIB2 units (with 2 additional LVDS connectors) - not only 4M, 8V or 8S. They can be found in 8W (A4/A5/Q5) and 4G (A6/A7) with slight differences in front panels used. Except TT's - it seems to be using the unit without these additional LVDS connectors.
The main concern for using Audi units for my purposes is that 8V1 units use 7" display with 800x480 resolution and all others (except TT - it is unique tho) use 8.25" display with 1024x480 resolution. So, eventually, if I chose to use for my conversion one of the 81A/4G/8W/4M units I have to get myself a second display with 1024x480 resolution - just to start firmware update. And only after MMX part of update succeeds I will be able to use my OEM 7" 800x480 display. That's why it is not a good idea to go with Audi units.
From the other side, the common thing between all Audi units is that they all are built on HW with numbers 050-055 which is the same as all other units 3G0/3V0/5F0 (starting from p/n with letter C and higher). So, firmware from 3G0/3V0 should fit 8V/8S/81A/4G/8W/4M units and vice versa. Theoretically, of course.
That's why I find this way most appropriate for my purposes - as long as I don't have any equipment that uses these two additional LVDS I would be comfortable with unit not having them.

**Reply by harman-f** [reply]
Resolution on these ports is software not hardware. So they are all good.

**Reply by em1ter** [reply]
Yes, that's right. But how does one change the resolution without having the unit reflashed?
For example, let's pretend I have purchased a 8W unit (being cheapest on the market). I want to flash it into 8V1 unit. I connect it to my car with 7" screen, turn on and see... Right, the only thing I see is blank screen. As far as I know, there will be no chance to enter red/green menu and make some settings to launch M.I.B tool, parse root password and connect to the unit with telnet, run update etc without the screen working. And eventually - no chance to change the resolution. But, all my knowledge is based on what I have read in the web as long as I have no practical experience with MIB2 HIGH, only some experience with MIB2 STD.
If you are aware of how to make MHI2 units that run 1024x480 display by default function properly on 800x480 display with just basic tools (USB ethernet adapter, few SD cards, laptop with ODIS and VAS5054) would you please share this knowledge?
Thank you.

**Comment by Wh1terat** [comment]
Sorry to dig up an old discussion.
@em1ter Did you have any issues in the end with resolution in order to flash AU37x  ?
(I need an 8V1 but currently 81A are cheapest)

**Comment by You553ph** [comment]
![IMG_3759](https://github.com/user-attachments/assets/7df821b7-60ff-4d6d-9806-f5bfbc6480a0)

**Comment by You553ph** [comment]
![IMG_3760](https://github.com/user-attachments/assets/76192560-e059-46dd-905e-8c305c8d4037)
https://mibwiki.one/doc/between-brands-g1g44zDQBW
